README.md

```diff
@@ -0,0 +1,188 @@
+# Enumerable
+## Getting Started
+This repository includes files ruby code on Enumerable and 3 related methods any , filter all:
+
```

Comment:
- [optional] I am not too sure if you actually meant to put this description here, because I felt you're repeating the same header you have written in the body of the README file. Kindly remove it. ℹ️